common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -178,8 +170,12 @@ public HelloCache getHelloCache() {
     }
 
     @VisibleForTesting
-    public String getActiveBrokerPackageName() {
-        return new BrokerValidator(mApplicationContext).getCurrentActiveBrokerPackageName();
+    public String getActiveBrokerPackageName() throws IllegalStateException{
+        try {
+            return new BrokerValidator(mApplicationContext).getCurrentActiveBrokerPackageName();
+        } catch (final ClientException e) {
```

Comment:
Is ClientException the only type of exception to catch here?

Reply:
yes, we are catching the exception in getCurrentActiveBrokerPackageName